lib/WP_Auth0_UsersRepo.php

```diff
@@ -132,9 +132,8 @@ public function find_auth0_user( $id ) {
 
 		$users = get_users( $query );
 
-		if ( $users instanceof WP_Error ) {
-			WP_Auth0_ErrorLog::insert_error( __METHOD__ . ' => get_users() ', $users->get_error_message() );
-
+		if ( $users === [] ) {
```

Comment:
I'm also curious about this change. Where was the default changed?

Reply:
No defaults were changed to my knowledge, the usage of the WP get_users function has remained unchanged since 3.1 as far as I can tell. I think we just made a mistake in writing our error handling here.

In our usage, [get_users()](https://core.trac.wordpress.org/browser/tags/5.6/src/wp-includes/user.php#L753), which is a wrapper for [WP_User_Query()->get_results()](https://developer.wordpress.org/reference/classes/wp_user_query/#return-values), will only ever return an array. The array returned will be empty if there were no results. 🤷‍♂️